src/OpenIddict.EntityFramework/OpenIddictDbContext.cs

```diff
@@ -150,63 +150,78 @@ public class OpenIddictDbContext<TUser, TRole, TApplication, TAuthorization, TSc
         protected override void OnModelCreating(ModelBuilder builder) {
             base.OnModelCreating(builder);
 
+            // Rename Identity tables.
+            builder.Entity<TUser>(e => e.ToTable("OpenIddictUsers"));
+            builder.Entity<TRole>(e => e.ToTable("OpenIddictRoles"));
+            builder.Entity<IdentityUserClaim<TKey>>(e => e.ToTable("OpenIddictUserClaims"));
+            builder.Entity<IdentityRoleClaim<TKey>>(e => e.ToTable("OpenIddictRoleClaims"));
+            builder.Entity<IdentityUserRole<TKey>>(e => e.ToTable("OpenIddictUserRoles"));
+            builder.Entity<IdentityUserLogin<TKey>>(e => e.ToTable("OpenIddictUserLogins"));
+            builder.Entity<IdentityUserToken<TKey>>(e => e.ToTable("OpenIddictUserTokens"));
+
             // Warning: optional foreign keys MUST NOT be added as CLR properties because
             // Entity Framework would throw an exception due to the TKey generic parameter
             // being non-nullable when using value types like short, int, long or Guid.
 
             // Configure the TApplication entity.
-            builder.Entity<TApplication>(entity => {
-                entity.HasKey(application => application.Id);
+            builder.Entity<TApplication>(e => {
+                e.HasKey(app => app.Id);
```

Comment:
When you contribute to a project, you MUST conform to its coding style and avoid changing things that don't need to be changed. Revert that.